t/08-autotest.t

```diff
@@ -84,6 +86,34 @@ is($died,      0, 'start+next+start should not die');
 is($completed, 1, 'start+next+start should complete');
 @sent = [];
 
+my $ctx = OpenQA::Test::Context->new();
+stderr_like(
+    sub {
+        autotest::loadtest("tests/context.pm", name => 'alt_name', context => $ctx);
```

Comment:
I think _'name'_ should be a context attribute, and not outside of it - otherwise it's not clear if _'name'_ is an attribute of the **test** or the **context**.

Aside: @coolo wdyt about allowing 'everything' to be passed into loadtest?

Reply:
It is the test's display name.

It is also the LTP test name, but they could be different.

Reply:
>  i'm not following what a freedom/empty implementation of context class could lead to

@foursixnine wanted type checking? It should stop some irritating Perl errors by making sure you have passed the right thing to the test. I can't think of any methods to add to it at this time though.